Source/HTTP/Android/http_android.cpp

```diff
@@ -3,13 +3,77 @@
 #include "pch.h"
 #if ANDROID_API
 #include <httpClient/httpClient.h>
+#include "android_http_request.h"
+
+void Internal_HCHttpCallInitialize(void* context)
+{
+    HttpRequest::InitializeJavaEnvironment(reinterpret_cast<JavaVM*>(context));
```

Comment:
>HttpRequest::InitializeJavaEnvironment(reinterpret_cast<JavaVM*>(context)); [](start = 4, length = 75)

should only do this if `HC_PLATFORM == HC_PLATFORM_ANDROID` and assert context is not null on other platforms you should `assert(context == nullptr);`